### DISCUSSION TITLE
Problem with tgslurmbot?

### DISCUSSION THREAD

**Discussion by hariseldon99** [CLIP-HPC/goslmailer#30] [General]
Hi,

Using version 2.7.1 from releases.

Trying to test tgslurmbot in [this fake docker cluster](https://hub.docker.com/r/giovtorres/slurm-docker-cluster) before putting it in my real one, but I am encountering issues.

Mailprog in slurm.conf is set:

```bash
# grep -B 2 -A 2 -i mailprog /etc/slurm/slurm.conf
#
# COMMUNICATION
MailProg=/usr/local/bin/goslmailer
# TIMERS
SlurmctldTimeout=300
```

Here is my config for tgslurmbot.conf/goslmailer.conf (same file, symlinked)

```json
{
  "logfile": "/tmp/goslmailer.log",
  "debugconfig": true,
  "binpaths": {
    "sacct":"/usr/bin/sacct",
    "sstat":"/usr/bin/sstat"
  },
  "defaultconnector": "telegram",
  "connectors": {
    "telegram": {
      "name": "testbot",
      "url": "",
      "token": "Phake:OAUTH:Placeholder",
      "renderToFile": "no",
      "spoolDir": "/tmp/telegramgobs",
      "messageTemplate": "/etc/slurm/telegramTemplate.html",
      "useLookup": "no",
      "format": "HTML"
    }
  },
  "qosmap": {
    "elevated": 3600,
    "normal": 28800
  }
}
```
Here is the bot in telegram, seemingly working when I sent /start to it after running 

```bash
# tgslurmbot &
```

![Screenshot from 2023-02-25 23-07-45](https://user-images.githubusercontent.com/10812394/221371576-1a8c321c-bfcc-46b3-be20-d177bf8880bd.png)

I submitted [this test code](https://github.com/hariseldon99/buparamshavak/blob/main/shavak_root/usr/local/share/examples/mpi_runs/hello_mpi.c) to slurm with sbatch: 

```bash
#!/bin/bash

#SBATCH --job-name=hello_mpi
#This sets the name of the job

#SBATCH --ntasks=1
#This sets the number of processes to 4. Change if needed

#SBATCH --cpus-per-task=1

#SBATCH --time=00:05:00

#SBATCH --qos=normal

#SBATCH --mail-type=ALL
#SBATCH --mail-user=telegram:5545394160

mpirun -np ${SLURM_NTASKS} ./hello_mpi
```

Code ran correctly, as evidenced by this output

```bash
Hello world from processor c1, rank 0 out of 1 processors
```

But no response from the bot. Nada.

Slurm version is:

```bash
# sinfo --version
slurm 21.08.6
```

**Comment by pja237** [comment]
Investigated in #31
